lib/apis/v3/workloadendpoint.go

```diff
@@ -37,9 +37,9 @@ type WorkloadEndpoint struct {
 // WorkloadEndpointMetadata contains the specification for a WorkloadEndpoint resource.
 type WorkloadEndpointSpec struct {
 	// The name of the orchestrator.
-	Orchestrator string `json:"orchestrator,omitempty" validate:"omitempty,namespacedname"`
+	Orchestrator string `json:"orchestrator,omitempty" validate:"omitempty,namespacedName"`
```

Comment:
Just name now I think.  i don't think we need this namespaced (although theoretically it shouldn't matter)